src/main/java/com/googlecode/download/maven/plugin/internal/WGet.java

```diff
@@ -73,7 +74,7 @@
  * @author Marc-Andre Houle
  * @author Mickael Istria (Red Hat Inc)
  */
-@Mojo(name = "wget", defaultPhase = LifecyclePhase.PROCESS_RESOURCES, requiresProject = false)
+@Mojo(name = "wget", defaultPhase = LifecyclePhase.PROCESS_RESOURCES, requiresProject = true)
```

Comment:
@leonard84 sorry, missed this thing at the initial CR.
Could we avoid requiring a project?

Reply:
Why? and not really `if (runOnlyAtRoot && !project.isExecutionRoot())`

Reply:
@leonard84 uff, it's already set to `true` by default (https://maven.apache.org/plugin-tools/maven-plugin-annotations/apidocs/org/apache/maven/plugins/annotations/Mojo.html#requiresProject--) so you just made it explicit. Nothing to worry about